app/controllers/games_controller.rb

```diff
@@ -16,15 +17,11 @@ def new
 
   def create
     @game = Game.new(game_params)
-
     @game.white_player_id = current_user.id
     @game.black_player_id = current_user.id
-    @game.game_status = "Unknown"
 
     if @game.save
-      puts "Save Successful"
-      render :show, status: :created
-      #redirect_to @game, notice: 'Game was successfully created.'
+      redirect_to @game, status: :created, notice: 'Game was successfully created.'
```

Comment:
Something may be wrong with the route here because when I ran the server and clicked through things it didn't automatically redirect after creating game though this can be fixed later. 
![youarebeingredirected](https://user-images.githubusercontent.com/22229082/28446551-7d31dae0-6d99-11e7-9521-581ad9352846.png)